packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -65,6 +66,43 @@ def vip_app(container: marathon.Container, network: marathon.Network, host: str,
         raise AssertionError('Unexpected network: {}'.format(network.value))
 
 
+def pod_vip_app(network: marathon.Network, host: str, vip: str):
```

Comment:
can we have a parameter to take in the network name? The parameter comes into play only if the mode is `USER`. It can default to `dcos`. Thinking about cases where we would like to install some test networks or we want to test against IPv6 networks going forward.

Reply:
This call for another PR